### PR TITLE
For x264 : Replace git repository by bz2 file and avoid to download it each time

### PR DESCRIPTION
**Pull request by GuillaumeSmaha**
Better than download each time ... but we have to think to delete the file downloaded if we want to update.

### CODE REVIEW

**Comment by Dr-Bean** [comment]
Better yet, just straight up pin the version. Doing that also prevents hammering, but additionally, it ensures correct downloads via checksumming and repeatability over multiple build runs because we can be sure that the version used is the same each time.

If you're concerned about cleaning up, just periodically delete the contents of $distrib_dir and redownload (e.g. run a `make downloads`)

**Comment by GuillaumeSmaha** [comment]
Yes this is right, I will update to x264-snapshot-20160517-2245-stable.tar.bz2

**Comment by GuillaumeSmaha** [comment]
@Dr-Bean it's done

File: cross/x264/Makefile [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
-PKG_DOWNLOAD_METHOD = git
-PKG_GIT_HASH = stable
-PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
-PKG_DIST_SITE = https://git.videolan.org/git/x264.git
-PKG_DIR = $(PKG_NAME)-git$(PKG_GIT_HASH)
+PKG_VERS = snapshot-20160517-2245-stable
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://ftp.videolan.org/pub/videolan/x264/snapshots
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)/
```
Comment (Dr-Bean) [inline cross/x264/Makefile]: Trailing slash should be unnecessary

**Comment by Dr-Bean** [comment]
Other than the one comment, yup :+1:

**Comment by GuillaumeSmaha** [comment]
@Dr-Bean Done :)

**Pull request merged**